x-pack/plugins/lens/public/editor_frame_service/service.tsx

```diff
@@ -59,6 +61,21 @@ export class EditorFrameService {
   private readonly datasources: Array<Datasource | Promise<Datasource>> = [];
   private readonly visualizations: Array<Visualization | Promise<Visualization>> = [];
 
+  private async documentToExpression(doc: Document) {
```

Comment:
I'd appreciate a comment here explaining how this is intended to be used by the embeddable